src/pages/JobOpeningPage/index.js

```diff
@@ -49,6 +49,17 @@ const useStyles = makeStyles((theme) => ({
       borderRadius: 5,
     },
   },
+  labelChip_techstack: {
+    margin: theme.spacing(1),
+    marginLeft: theme.spacing(2),
+    "& .MuiChip-root": {
+      marginRight: theme.spacing(1),
+      color: "#FFFFF",
+      background: "#ffce47",
+      fontWeight: 700,
+      borderRadius: 100,
+    },
```

Comment:
Instead of doing this and nesting css inside another one.
Better is to create a new class make css styles for that separetely and reference it in the react

Reply:
The original code was also having the same nesting so I don't change it! Ok  So I will make a new class for the tech stack and declare it below the CSS for `internship` box

Reply:
```diff
-    "& .MuiChip-root": {
-      marginRight: theme.spacing(1),
-      color: "#FFFFF",
-      background: "#ffce47",
-      fontWeight: 700,
-      borderRadius: 100,
-    },
```

```diff
+ someclass: {
+      marginRight: theme.spacing(1),
+      color: "#FFFFF",
+      background: "#ffce47",
+      fontWeight: 700,
+      borderRadius: 100,
+ }
```

and refer that class in here
```diff
- <Chip size="small" label="JS" />
+ <Chip size="small" label="JS" className={classes.someClass} />
```

Reply:
Ok Got it